src/web3/contracts.ts

```diff
@@ -32,12 +34,42 @@ let contractKit: ContractKit | undefined
 
 const initContractKitLock = new Lock()
 
+// @ts-ignore
+async function testSigning(newWallet: CapsuleWallet) {
```

Comment:
initially this looked like a test- perhaps a name like signTestTransaction could clarify this purpose more (and maybe a comment explaining this)

Also very small nit, but `const PAYLOAD ={...]`